Rubberduck.Core/AutoComplete/Service/SelfClosingPair.cs

```diff
@@ -1,7 +1,18 @@
-﻿namespace Rubberduck.AutoComplete.Service
+﻿using Rubberduck.VBEditor;
+using System;
+
+namespace Rubberduck.AutoComplete.Service
 {
-    public class SelfClosingPair
+    public class SelfClosingPair : IEquatable<SelfClosingPair>
     {
+        [Flags]
+        public enum MatchType
+        {
+            NoMatch = 0,
+            OpeningCharacterMatch = 1,
+            ClosingCharacterMatch = 2,
+        }
```

Comment:
note to self: enum is a relic of an aborted attempt at the char-lookup mechanism... remove this.